.github/workflows/check-source-signatures.yml

```
@@ -45,14 +48,22 @@ jobs:
       - name: Get changed packages
         run: |
           # Find the packages that have been modified in the current PR. They will be of the form '/path/to/SPECS/<pkgname>/**/.*', and we want to extract
-          # the package name (ie the folder inside ./SPECS).
-          changed_pkgs=$(git diff-tree --diff-filter=d --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }} | { grep "SPECS/.*" || test $? = 1; } | sed -n 's#SPECS/\([^/]*\)/.*#\1#p' | sort -u | xargs)
-          changed_pkgs_extended=$(git diff-tree --diff-filter=d --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }} | { grep "SPECS-EXTENDED/.*" || test $? = 1; } | sed -n 's#SPECS-EXTENDED/\([^/]*\)/.*#\1#p' | sort -u | xargs)
+          # the package's directory name (ie the folder inside ./SPECS).
+          changed_pkg_dirs=$(git diff-tree --diff-filter=d --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }} | { grep "${{ matrix.specs-dir }}/.*" || test $? = 1; } | sed -n 's#SPECS/\([^/]*\)/.*#\1#p' | sort -u | xargs)
```

```suggestion
          changed_pkg_dirs=$(git diff-tree --diff-filter=d --no-commit-id --name-only -r ${{ env.base_sha }} ${{ github.sha }} | { grep "${{ matrix.specs-dir }}/.*" || test $? = 1; } | sed -n 's#${{ matrix.specs-dir }}/\([^/]*\)/.*#\1#p' | sort -u | xargs)
```

I still like the `grep -oP` version better to drop `sed` but as long as it works, I'm good.:)

---

Oh, I lost the grep in a fixup. I didn't mind it either. I'll put it back.

---

Oh, I remember why I removed it now... `grep -oP "(?<=${{ matrix.specs_folder }}/)([^/]+)"` But I've put it back because I hate reading `sed`.